tests/telemetry/events/dhcp_relay_events.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python3
+
+import logging
+import time
+from run_events_test import run_test
+from tests.common.utilities import wait_until
+
+logger = logging.getLogger(__name__)
+tag = "sonic-events-dhcp-relay"
+
+interface = ""
```

Comment:
>interface [](http://example.com/codeflow?start=0&length=9)

Could you prevent using file global variables? I see the are used in test_event (direct or indirectly). So I think at least lambda and function currying will help.

Reply:
Removed dependency on global variables